lib/ssh-libssh.c

```diff
@@ -1586,7 +1586,7 @@ static CURLcode myssh_statemach_act(struct connectdata *conn, bool *block)
             return CURLE_BAD_DOWNLOAD_RESUME;
           }
         }
-        /* Does a completed file need to be seeked and started or closed ? */
```

Comment:
I know seeked isn't the correct spelling, but in this case I'm ready to give it the benefit of being wrong since it's referring to the specific `seek` call.. The current spelling, in all it's typoness, convey meaning.

Reply:
Reasonable, I'll revert it tomorrow.